src/BenchmarkDotNet/Reports/SummaryStyle.cs

```diff
@@ -70,6 +72,7 @@ public bool Equals(SummaryStyle other)
                    && PrintUnitsInContent == other.PrintUnitsInContent
                    && PrintZeroValuesInContent == other.PrintZeroValuesInContent
                    && Equals(SizeUnit, other.SizeUnit)
+                   && Equals(SizeUnit, other.CodeSizeUnit)
```

Comment:
Typo? Should be Equals(**Code**SizeUnit, other.CodeSizeUnit).

Reply:
Nice! Will you open PR or I should add a bug fix to this branch?

Reply:
@YegorStepanov I think it makes more sense if you add the bugfix.

Reply:
@mawosoft I can't add commit to here. Github is inflexible. If you or Adam will not have time to add a bugfix. I will create it with another PR later today. (It will be the 5 PR in 3 days :))